### PR TITLE
Added HTTP Proxy section

### PR DESCRIPTION
**Pull request by jaleksynas**
HTTP proxy seems to be a pretty common issue, I added a section near firewall issues to assist those in need.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -243,6 +243,17 @@ Detailed instructions about how to set up `deliver` and `fastlane` in `Jenkins`
 DELIVER_ITMSTRANSPORTER_ADDITIONAL_UPLOAD_PARAMETERS="-t DAV" deliver
 ```
 
+## HTTP Proxy
+iTunes Transporter is a Java application bundled with Xcode.  In addtion to utilizing the `DELIVER_ITMSTRANSPORTER_ADDITIONAL_UPLOAD_PARAMETERS="-t DAV"`, You need to configure the transporter application to use the proxy independently from the system proxy or any environment proxy settings.  You can find the configuration file within Xcode:
```
Comment (KrauseFx) [inline README.md]: There are 2 spaces after `Xcode.`

Comment (KrauseFx) [inline README.md]: `You` after `,` shouldn't be upper case, right?

Comment (KrauseFx) [inline README.md]: Or replace `,` with `.`

Comment (KrauseFx) [inline README.md]: 2 spaces after `settings.`

File: README.md [hunk 2]
```diff
@@ -243,6 +243,17 @@ Detailed instructions about how to set up `deliver` and `fastlane` in `Jenkins`
...
+
+```bash
+TOOLS_PATH=$( xcode-select -p )
+REL_PATH='../Applications/Application Loader.app/Contents/itms/java/lib/net.properties'
+echo $TOOLS_PATH/$REL_PATH
+```
+
+Add necessary proxy configuration values to the net.properties according to [Java Proxy Configuration](http://docs.oracle.com/javase/8/docs/technotes/guides/net/proxies.html)
```
Comment (KrauseFx) [inline README.md]: Please add `.` after the `)`

**Comment by KrauseFx** [comment]
This is a great addition, thanks. I added some small comments

**Comment by jaleksynas** [comment]
Updated according to comments.

File: README.md [hunk 3]
```diff
@@ -243,6 +243,17 @@ Detailed instructions about how to set up `deliver` and `fastlane` in `Jenkins`
 DELIVER_ITMSTRANSPORTER_ADDITIONAL_UPLOAD_PARAMETERS="-t DAV" deliver
 ```
 
+## HTTP Proxy
+iTunes Transporter is a Java application bundled with Xcode. In addtion to utilizing the `DELIVER_ITMSTRANSPORTER_ADDITIONAL_UPLOAD_PARAMETERS="-t DAV"`, you need to configure the transporter application to use the proxy independently from the system proxy or any environment proxy settings. You can find the configuration file within Xcode:
+
+```bash
+TOOLS_PATH=$( xcode-select -p )
+REL_PATH='../Applications/Application Loader.app/Contents/itms/java/lib/net.properties'
+echo $TOOLS_PATH/$REL_PATH
```
Comment (samrobbins) [inline README.md]: This line requires quotes to function correctly.

**Comment by jaleksynas** [comment]
Corrected.

**Comment by KrauseFx** [comment]
Thanks @jaleksynas for contributing :+1:

**Pull request merged**